modules/python/init.zsh

```diff
@@ -160,10 +160,11 @@ if (( $#commands[(i)pip(|[23])] )); then
         || ! -s "$cache_file" ]]; then
     # pip is slow; cache its output. And also support 'pip2', 'pip3' variants
     $pip_command completion --zsh \
-      | sed -e "s|compctl -K [-_[:alnum:]]* pip|& pip2 pip3|" >! "$cache_file" 2> /dev/null
+      | sed -e "s|\(compctl -K [-_[:alnum:]]*\) pip|\1 pip pip2 pip3|" >! "$cache_file" 2> /dev/null
```

Comment:
What does this regex even do?

Reply:
Replaces the line `compctl -K _pip_completion pip` with `compctl -K _pip_completion pip pip2 pip3` in the output buffer of `pip completion --zsh`.

This snippet will probably make it a bit more obvious:

```shell
~❯❯❯ diff <(pip completion --zsh) <(pip completion --zsh | sed -e "s|\(compctl -K [-_[:alnum:]]*\) pip|\1 pip pip2 pip3|")
11c11
< compctl -K _pip_completion pip
---
> compctl -K _pip_completion pip pip2 pip3
```